### PR TITLE
tell user config setting is unchanged

### LINKED ISSUE #197: tell user if setting already set
when setting settings, such as `config channel` then don't set them, if they're already set

so, instead of
```
config c sed on
config c sed on
```
resulting in 
```
[Config] Config 'sed' set to True
[Config] Config 'sed' set to True
```
it should be more like
```
[Config] Config 'sed' set to True
[Config] Config 'sed' was already set to True !
```

### PR DESCRIPTION
**Pull request by fosslinux**
Closes #197 

Review comments just for @jesopo things i wasnt completely sure about below :)

### CODE REVIEW

File: modules/config.py [hunk 1]
```diff
@@ -231,6 +237,9 @@ def config(self, event):
             elif result.result == ConfigResults.Removed:
                 event["stdout"].write("Unset setting '%s'%s" %
                     (setting.lstrip("-"), for_str))
+            elif result.result == ConfigResults.Unchanged:
+                event["stdout"].write("Config '%s'%s unchanged" %
```
Comment (fosslinux) [inline modules/config.py:241]: Should I have this as `'%s'` or `'%s'%s`

Comment (jesopo) [reply modules/config.py:241]: `'%s'%s` is correct!

File: modules/config.py [hunk 2]
```diff
@@ -121,9 +123,13 @@ def _config(self, export_settings, target, setting, value=None):
                 raise ConfigInvalidValue(str(e))
 
             if not validated_value == None:
-                target.set_setting(setting, validated_value)
+                existing_value = target.get_setting(setting, None)
                 formatted_value = setting_object.format(validated_value)
```
Comment (jesopo) [inline modules/config.py]: can this line also be moved to after we've decided the value has changed?

Comment (fosslinux) [reply modules/config.py]: yes it can, good pickup

**Comment by jesopo** [comment]
thanks!

**Pull request merged**